src/api/team-membership/content-types/team-membership/schema.json

```diff
@@ -0,0 +1,32 @@
+{
+  "kind": "collectionType",
+  "collectionName": "team_memberships",
+  "info": {
+    "singularName": "team-membership",
+    "pluralName": "team-memberships",
+    "displayName": "TeamMembership",
+    "description": ""
+  },
+  "options": {
+    "draftAndPublish": false
+  },
+  "pluginOptions": {},
+  "attributes": {
+    "user": {
```

Comment:
I think this should be a required field.

Reply:
Okay, i will do that now

Reply:
Done now